src/main/webapp/src/components/Basket.js

```diff
@@ -0,0 +1,113 @@
+import React from "react";
+import axios from "axios";
+import {Button} from "antd";
+import {DeleteFilled} from "@ant-design/icons";
+import '../styles/Basket.css'
+import '../styles/NotFoundPage.css'
+
+class Basket extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            phones: [],
+            numberItems: 0,
+            totalPrice: 0,
```

Comment:
keep the whole basket in state. also better to calculate totalPrice on server. not on client

Reply:
+